README.md

```diff
@@ -1,34 +1,89 @@
-# kafka-connect-datagen
+# Overview
 
-`kafka-connect-datagen` is a Kafka Connect connector for generating mock data, not suitable for production
+`kafka-connect-datagen` is a Kafka Connect connector for generating mock data.
+It is available in [Confluent Hub](https://www.confluent.io/connector/kafka-connect-datagen/).
+It is not suitable for production.
 
-## Local
+# Confluent Platform running on local install
+
+## Install connector from Confluent Hub
+
+You may install the `kafka-connect-datagen` connector from [Confluent Hub](https://www.confluent.io/connector/kafka-connect-datagen/).
+
+```bash
+confluent-hub install --no-prompt confluentinc/kafka-connect-datagen:0.1.0
+```
+
+## Build connector from latest code
+
+Alternatively, you may build and install the `kafka-connect-datagen` connector from latest code.
 
 ```bash
 git checkout v0.1.0
-confluent destroy
 mvn clean compile package
 confluent-hub install --no-prompt target/components/packages/confluentinc-kafka-connect-datagen-0.1.0.zip
+```
+
+## Run connector in local install
+
+Here is an example of how to run the `kafka-connect-datagen` on a local install:
+
+```bash
+confluent destroy
 confluent start connect
 sleep 15
 confluent config datagen -d ./connector_datagen.config
 #confluent config datagen -d ./connector_datagen.custom.config
 sleep 5
```

Comment:
Would it be possible to remove the call to `sleep` here and maybe just have a note about how it may take a few seconds for the `confluent status connectors` command to show the new connector?

Reply:
@C0urante : I removed all the sleeps